Bottom: 7574733bce532f59e4774d9f673cce4b9f5451f9
Top:    68e89cf1610b4f73ee5afa33d14d24e0ce412529
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-07-26 00:14:22 -0700

Refresh of rst-afib

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 3ff79d8..6339c47 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1194,8 +1194,8 @@ int raccept(int socket, struct sockaddr *addr, socklen_t *addrlen)
 	rs_save_conn_data(new_rs, creq);
 	param = new_rs->cm_id->event->param.conn;
 	rs_format_conn_data(new_rs, &cresp);
-	param->private_data = &cresp;
-	param->private_data_len = sizeof cresp;
+	param.private_data = &cresp;
+	param.private_data_len = sizeof cresp;
 	ret = rdma_accept(new_rs->cm_id, &param);
 	if (!ret)
 		new_rs->state = rs_connect_rdwr;
@@ -1264,8 +1264,8 @@ do_connect:
 		memset(&param, 0, sizeof param);
 		creq = (void *) &cdata + rs_conn_data_offset(rs);
 		rs_format_conn_data(rs, creq);
-		param->private_data = (void *) creq - rs_conn_data_offset(rs);
-		param->private_data_len = sizeof(*creq) + rs_conn_data_offset(rs);
+		param.private_data = (void *) creq - rs_conn_data_offset(rs);
+		param.private_data_len = sizeof(*creq) + rs_conn_data_offset(rs);
 		param.flow_control = 1;
 		param.retry_count = 7;
 		param.rnr_retry_count = 7;
